content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst

```diff
@@ -1,418 +1,171 @@
+documentation page.
+
+.. _inventory/warehouses_storage/removal-config:
+
+Configuration
+=============
+
+Removal strategies are set on either the product category or storage location.
+
+Configure removal strategies on the location by going to :menuselection:`Inventory --> Configuration
+--> Locations`, and selecting the desired location. On the location form, choose a removal strategy
+from the :guilabel:`Removal Strategy` field's drop-down menu options.
+
+.. important::
+   To set a removal strategy on a location, the :guilabel:`Storage Locations` and
+   :guilabel:`Multi-Step Routes` settings **must** be enabled in :menuselection:`Inventory -->
+   Configuration --> Settings`.
 
-Lot `00001` entered the stock on May 23, lot `00002` on May 25, and lot `00003` on June 1. A
-customer orders six boxes on June 11.
+   These features are **only** necessary when setting the removal strategy on a location.
 
-Using the :abbr:`FIFO (First In, First Out)` removal strategy, a transfer request will pick the five
-boxes from lot `00001` first, and then from the boxes in lot `00002`, since lot `00001` entered the
-stock first. The box from lot `00002` is taken next because it has the oldest receipt date after lot
-`00001`.
+Configure removal strategies on product categories by going to :menuselection:`Inventory -->
+Configuration --> Product Categories` and selecting the intended product category. Next, choose a
+removal strategy from the :guilabel:`Force Removal Strategy` drop-down menu options.
 
-.. image:: removal/fifo-nails-picking.png
-   :align: center
-   :alt: The detailed operations for the transfer shows the nail lots to be removed.
-
-Last In, First Out (LIFO)
--------------------------
-
-Similar to the :abbr:`FIFO (First In, First Out)` method, the :guilabel:`Last In, First Out (LIFO)`
-removal strategy moves products based on the date they entered a warehouse's stock. Instead of
-removing the oldest stock on-hand, however, it targets the **newest** stock on-hand for removal.
-
-Every time an order for products with the :abbr:`LIFO (Last In, First Out)` method is placed, a
-transfer is created for the lot/serial number that has most recently entered the stock (the **last**
-lot/serial number that entered the warehouse's inventory).
+.. important::
+   When there are different removal strategies applied on both the location and product category for
+   a product, the value set on the :guilabel:`Force Removal Strategy` field set on a
+   :guilabel:`Product Category` form is applied as top priority.
 
-.. warning::
-   In many countries, the :abbr:`LIFO (Last In, First Out)` removal strategy in banned, since it can
-   potentially result in old, expired, or obsolete products being delivered to customers.
+   .. image:: removal/navigate-location-category.png
+      :align: center
+      :alt: Change the Force Removal Strategy for either the Product Categories or Locations.
```

Comment:
Was this image always set in the admonition like this or did it move since last review?
Seems to fit better right after line 44, up to you 🙂